Translations-Guide.md

```diff
@@ -3,17 +3,47 @@
 Thank you so much for your interest in translating FreeCodeCamp. Reading this document is recommended for anyone who wishes to participate in this collective effort to bring FreeCodeCamp to more and more people all around the world.
 
 ## 1. Chat room
-We recommend you to create a chat room in gitter.im exclusive for the language that you're working on, communicate there in that language, discuss the current the status of the project and coordinate efforts. Help regarding the technical process should also be provided in the chat room. Many people want to collaborate, but get intimidated by the learning curve of Git. While it's not difficult to get started, it's always nice to have someone to help you if you get stuck. Many users go to the official translators chat room of FreeCodeCamp at [**`[FreeCodeCamp/Translators]`**](https://gitter.im/FreeCodeCamp/Translators) . You can go there to recruit new translators for your language every once in a while.
+
+
+Please join the gitter channel [https://gitter.im/FreeCodeCamp/Translators]
+and also join the channel for translators of your specific language 
+(if it exists):
+
+* Amharic [FreeCodeCamp/TranslationAmharic](https://gitter.im/FreeCodeCamp/TranslationAmharic)
+* Arabic [FreeCodeCamp/TranslationArabic](https://gitter.im/FreeCodeCamp/TranslationArabic)
+* Bangla: [FreeCodeCamp/TranslationBangla](https://gitter.im/FreeCodeCamp/TranslationBangla)
```

Comment:
> Personally for translation channels I prefer transliteration of the language name to the english alphabet (e.g Deutsch instead of German, Espanol instead of Spanish and so on).

I agree to this. Most translators will find rooms from the list of rooms so I guess its Okay this way?